scripts/globals/abilities/steal.lua

```diff
@@ -71,9 +73,10 @@ ability_object.onUseAbility = function(player, target, ability, action)
     local stealChance = 50 + stealMod * 2 + thfLevel - target:getMainLvl()
 
     stolen = target:getStealItem()
-    if (target:isMob() and math.random(100) < stealChance and stolen ~= 0) then
-        if (checkThfAfQuest(player, target) == true) then
-            stolen = 4569
+    if target:isMob() and math.random(100) < stealChance and stolen ~= 0 then
+        -- This guarantees item if quest is flagged
+        if checkThfAfQuest(player, target) == true then
```

Comment:
I just checked on 3 different characters, one that just flagged the quest, one that finished it ages ago, and one that's never flagged it, all of them _only_ stole Quadav Stew from Zircon and Silver quadavs in Beadeaux (others untested, they didn't have any other stealable items listed other than the stew). Either they only have Quadav Stew, or the silver beastcoin is not stealable from them.

Is this check because the steal pool is wrong/missing, or just that one line in ffxiclopedia that _claims_ that "if you are on THF quest, you will always get a stew"?

Reply:
Was the stealing of stew on retail?  I always assumed they could drop both like the wikis say.

Yes that's what this check does.  It was already there, I just cleaned it up a little.

Reply:
Yes, it was retail. As far as I can tell (without doing hundreds of samples) silver beastcoin is probably not a steal drop on the mobs that you can steal the Stew from in Beadeaux

Reply:
If that's the case we could rip out this whole section and remove the drops in the SQL.  I'm trying it on retail now and I'm 5/5 on stew as well.

Reply:
If you want to rip out the drops I'd be fine with that, but we could punt it for a different PR as well. As you said this isn't something this PR introduced. Let me know what you want to do!

Reply:
If I end up doing it, I'd rather do it in a different PR.

I wonder if they changed it at some point.  The horror stories about the drop rate seem to stop about 2010.